crates/env/src/call/call_builder.rs

```diff
@@ -30,6 +30,7 @@ use core::marker::PhantomData;
 
 /// The final parameters to the cross-contract call.
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Can you add a short command explaining why this is okay?

Reply:
Ah this was leftover code, thanks for spotting.